src/routes/jira/jira-get-connected-repos.ts

```diff
@@ -54,7 +54,7 @@ export const JiraGetConnectedRepos = async (
 				[Op.and]: [
 					{
 						repoName: {
-							[Op.like]: `%${repoName}%`
```

Comment:
Hi while you are here, can you also have a look at is this replacement vulnerable for the SQL injection attack? I suspect we should use something else instead of ``[Op.like]: `...${}...` ``